src/Routing/Router.php

```diff
@@ -594,6 +594,8 @@ public static function url($url = null, $full = false)
                 unset($url['_ssl']);
             }
 
+            $url = static::_applyUrlFilters($url);
```

Comment:
That was the case earlier too. If you see line 599 below only if `_name` was not set the params from current request are merged. So as far as urls with `_name` are concerned this change has no effect.